src/rpc/blockchain.cpp

```diff
@@ -1318,7 +1318,11 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  ],\n"
             "  \"headers\": xxxxxx,            (numeric) the current number of headers we have validated\n"
             "  \"bestblockhash\": \"...\",       (string) the hash of the currently best block\n"
-            "  \"difficulty\": xxxxxx,         (numeric) the current difficulty\n"
```

Comment:
This is a non-backwards compatible change. We can't remove fields as we don't know what someone might be using to parse the JSON output. Might cause their end to break.

Reply:
This was a bug fix of the rpc help. The JSON returned keys for "difficulty_pow" and "difficulty_pos" have not changed.
`   obj.pushKV("difficulty_pow",        nDiffPoW);
    obj.pushKV("difficulty_pos",        nDiffPoS);`

[current master](https://github.com/Veil-Project/veil/blob/6a1f9f8426b58d98aa3e55aa13e3c6464f336303/src/rpc/blockchain.cpp#L1366)

Reply:
@mimirmim yes; Code is correct; help doc was incorrect.  See below:

![image](https://user-images.githubusercontent.com/36988814/78185873-c11bbd80-7439-11ea-8120-1151b8fb0d84.png)

so it's only additions, not changes of existing.